examples/gem/WORKSPACE

```diff
@@ -3,6 +3,16 @@ local_repository(
     path = "../..",
 )
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
```

Comment:
note: we could introduce a `rules_ruby_dependencies`-style macro for users to call here instead, but I think with bzlmod nearly here we might not need it, WDYT?